### PR TITLE
Project pages: updates for geosmart

### PR DESCRIPTION
**Pull request by aaarendt**
This PR simplifies the structure of our Jupyter Book project pages. It is intended to support specific ways we plan to run GeoSMART, but also incorporates changes that we've been introducing to project structures in the past year. Some of these changes will eventually need to be added back into our hackweek website template.

### CODE REVIEW

**Comment by aaarendt** [comment]
@spestana can you assist in linking this project roadmap to your new project template? I also need help updating the roadmap in general.

File: book/img/projects-montage.png
Comment (jomey) [inline book/img/projects-montage.png:1]: Ideally for a image on the web, we should downscale them ahead of time and reduce the size. I imagine that this won’t need the current full resolution?

File: book/_toc.yml [hunk 1]
```diff
@@ -31,12 +31,11 @@ parts:
         - file: tutorials/example/tutorial-notebook
 - caption: Projects
   chapters:
-    - file: projects/list_of_projects
     - file: projects/index
       sections:
-        - file: projects/project_roles
         - file: projects/project_roadmap
         - file: projects/project_initialization
+    - file: projects/list_of_projects
```
Comment (NCristea) [inline book/_toc.yml]: Does this need to be aligned with the lines above?

**Review by NCristea — APPROVED** [review]
lgtm

**Review by friedrichknuth — APPROVED** [review]

File: book/img/projects-montage.png
Comment (scottyhq) [reply book/img/projects-montage.png:1]: We should have guidelines on file max size. Generally I've been thinking <5MB is fine. Two related issues to this on the template repo:
1.  a check for large files: https://github.com/uwhackweek/jupyterbook-template/issues/110
2. web-optimizations: https://github.com/uwhackweek/jupyterbook-template/issues/120

File: book/projects/index.md [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
 :class: bg-primary mb-1
-:width: 500px
+:width: 700px
 :align: center
 ```
 
-## Projects from Past Events
+## GeoSMART Hackweek Projects   
+
+The 2023 GeoSmart Hackweek will focus primarily on project work, with an emphasis on applications in Hydrology and Cyrosphere science. Project ideas will be shared in advance. Our goal is to have all participants connected to a project prior to the hackweek.
```
Comment (spestana) [inline book/projects/index.md:21]: A link to the project repo template could be useful here (note that the link in project_initialization is broken)

File: book/projects/project_roadmap.md [hunk 3]
```diff
@@ -13,25 +13,12 @@ Each project will progress through these stages:
...
 ### Day 1 of the Event
 
-We facilitate a project team formation session at the end of the first day of the event.
-
-{fa}`check,text-success mr-1` People with project ideas will make a 1-minute pitch to the group
-
-{fa}`check,text-success mr-1` People looking for a project are invited into small groups to learn more about existing ideas
-
-{fa}`check,text-success mr-1` UW eScience helpers and facilitators are available to help teams get clear on size and scope of the effort
+We facilitate a project team formation session on the first day of the hackweek. During this time we will finalize project goals and objectives. Hackweek organizers and facilitators will help each team set reasonable goals and develop a roadmap for task completion.
```
Comment (spestana) [inline book/projects/project_roadmap.md:21]: A link to the project repo template could be useful here (note that the link in project_initialization is broken)

**Review by spestana — APPROVED** [review]
This looks good as is, but I noted a couple of spots where links to the project template might help participants navigate. (note that the link in project_initialization.md is broken - perhaps we can fix that as part of this PR)

**Pull request merged**